test cases/python/8 different python versions/test.json

```diff
@@ -5,6 +5,7 @@
         { "val": null },
         { "val": "python2" },
         { "val": "python3" },
+        { "val": "python" },
```

Comment:
Why do we need to add another matrix case here? It should already be triggered 4 times by each of the existing matrix cases that don't pass an empty string.

Reply:
That is because the `py.exe` fallback skips it, but it should perform the test properly in CI, which has a python3.exe in the path, or when a machine file specifies the location of python.

Reply:
That could work. It does sound a bit more like a basic test than one that has to test the difference between python3 and python2 and pypy.